### PR TITLE
Feature/connecting to node

### PR DESCRIPTION
**Pull request by MarcRubio**

### CODE REVIEW

File: app/src/main/java/com/radixdlt/android/ui/dialog/InputNodeIPAddressDialog.kt [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+                if (v.customRadioButton.isChecked && !valid) {
+                    activity!!.toast("IP address is NOT valid")
+                    return@setOnClickListener
+                }
+
+//                QueryPreferences.setPrefRandomNodeSelection(
+//                    v.context,
+//                    v.randomRadioButton.isChecked
+//                )
+//                QueryPreferences.setPrefNodeIP(v.context, ipAddress)
```
Comment (laszlourszuly) [inline app/src/main/java/com/radixdlt/android/ui/dialog/InputNodeIPAddressDialog.kt]: Please consider removing commented out code blocks.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -28,7 +28,8 @@
   <string name="enter_password_activity_unlock_button">Unlock Wallet</string>
   <string name="enter_password_activity_creating_wallet_progress_dialog">Creating Wallet…</string>
   <string name="enter_password_activity_unlocking_wallet_progress_dialog">Unlocking Wallet…</string>
-  <string name="enter_password_activity_xml_universe">Connecting to universe&#160;</string>
+  <string name="enter_password_activity_xml_universe">Connecting to universe\u0020</string>
+  <string name="enter_password_activity_xml_specific_node">and node\u0020</string>
```
Comment (laszlourszuly) [inline app/src/main/res/values/strings.xml]: Don't change this now, but for future reference, maybe we could add a short comment clarifying that `\u0020` is the unicode character for a "single breakable space".

Also, is a breakable space we want here (`\u0020`) or the non-breakable counterpart (`\u00A0`). Depends on what we're concatenating with I guess.

File: app/src/main/java/com/radixdlt/android/ui/dialog/InputNodeIPAddressDialog.kt (hunk 1 above)
Comment (MarcRubio) [reply app/src/main/java/com/radixdlt/android/ui/dialog/InputNodeIPAddressDialog.kt]: ahh forgot about this, will clean up straight after merge!

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (MarcRubio) [reply app/src/main/res/values/strings.xml]: agree

**Pull request merged**